seneca/seneca.d.ts

```diff
@@ -3,229 +3,227 @@
 // Definitions by: Peter Snider <https://github.com/psnider/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/seneca
 
+declare function SNS(options?: SNS.Options): SNS.Instance;
```

Comment:
Seneca is node-only.

How should I place this inside the node module?
like this?
```
declare module "seneca" {
    function SNS(options?: SNS.Options): SNS.Instance;
    namespace SNS {
        ...
    }
    export = SNS;
}
```